docs/account.mdx

```diff
@@ -0,0 +1,533 @@
+to the target address inside the selected ParaTime.
+
+![code shell](../examples/account.13.y.in)
+
+![code](../examples/account.13.y.out)
+
+If no address is provided, the deposit will be made to the address
+corresponding to your consensus account inside the ParaTime.
+
+![code shell](../examples/account.14.y.in)
+
+![code](../examples/account.14.y.out)
+
+Currently, deposit transactions are free of charge, hence the `--gas-price 0`
+parameter to avoid spending unnecessary gas fees. Also, keep in
+mind that **deposit and withdrawal fees are always paid by your ParaTime
+account.** If it doesn't contain any ROSE, you will not able to cover the fees.
+
+You can also make a deposit to an account with arbitrary address inside a
+ParaTime. For example, let's deposit to some native address inside the
+ParaTime:
+
+![code shell](../examples/account.15.y.in)
+
+![code](../examples/account.15.y.out)
+
+Or to some address in the Ethereum format:
+
+![code shell](../examples/account.16.y.in)
+
+![code](../examples/account.16.y.out)
+
+:::info
+
+[Network, ParaTime and Account](#npa) selectors are available for the
+`account deposit` command.
+
+:::
+
+## Withdraw Tokens from the ParaTime {#withdraw}
+
+`account withdraw <amount> [to]` will withdraw funds from your ParaTime account
+to a consensus address:
+
+![code shell](../examples/account.17.y.in)
+
+![code](../examples/account.17.y.out)
+
+If the address is not provided, the address of the account inside ParaTime will be
+used as a consensus address:
+
+![code shell](../examples/account.18.y.in)
+
+![code](../examples/account.18.y.out)
+
+:::caution
+
+Withdrawal transactions are not free of charge and the fee will be deducted
+**from your ParaTime balance**.
+
+:::
+
+Similar to the [`account deposit`](#deposit) command, you can also specify an arbitrary
+Oasis address which you want to withdraw your tokens to.
+
+![code shell](../examples/account.19.y.in)
+
+![code](../examples/account.19.y.out)
+
+:::caution
+
+You cannot use the destination address of your `secp256k1` account or any other
+Ethereum-formatted address for the withdrawal, because this signature scheme is
+not supported on the consensus layer!
+
+:::
+
+:::info
+
+[Network, ParaTime and Account](#npa) selectors are available for the
+`account withdraw` command.
+
+:::
+
+## Delegate Tokens to a Validator {#delegate}
+
+To stake your tokens on the consensus layer, run
+`account delegate <amount> <to>`. This will delegate the specified amount of
+tokens to a validator.
+
+![code shell](../examples/account.20.y.in)
+
+![code](../examples/account.20.y.out)
+
+Once your tokens are staked, they are denoted as *shares* since the number of
+tokens will increase over time due to [positive interest rate][token-metrics].
+The number of shares on the other hand will remain constant, if your validator
+is not subject to [slashing]. Also, shares are always interpreted as a whole
```

Comment:
The _number_ of shares will remain constant even in case of slashing, it is only their _value_ that will decrease.